localstack/services/cloudformation/models/sqs.py

```diff
@@ -50,8 +49,11 @@ def _delete(resource_id, resources, *args, **kwargs):
             for queue in props["Queues"]:
                 try:
                     sqs_client.set_queue_attributes(QueueUrl=queue, Attributes={"Policy": ""})
-                except QueueDoesNotExist:
-                    LOG.debug("Queue resource was already deleted.")
+                except Exception as err:
+                    if "AWS.SimpleQueueService.NonExistentQueue" == err.response["Error"]["Code"]:
```

Comment:
nit: slightly simpler:
```
                    if "AWS.SimpleQueueService.NonExistentQueue" != err.response["Error"]["Code"]:
                        raise
```
Also, we could catch on `botocore.exceptions.ClientError`, to ensure that the `err.response` attribute is present.